torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -49,6 +55,82 @@ std::vector<bool> TensorDesc::findContiguous(
   return cont;
 }
 
+// Descriptor for chunk-ing an input tensor into subtensors
+// OR concat-ing an output tensor from subtensors
+struct PartitionDesc {
```

Comment:
Move only?

Reply:
Move + removed the chunk constructor, because it was unnecessary.